src/hooks/xcm/SubstrateApi.ts

```diff
@@ -98,7 +98,19 @@ class ChainApi {
       ss58Prefix,
     };
   }
-
+  public async evmTransferToParachain({
+    toPara,
+    recipientAccountId,
+    amount,
+    selectedToken,
+  }: {
+    toPara: number;
+    recipientAccountId: string;
+    amount: string;
+    selectedToken: Asset;
+  }): Promise<string> {
+    return '';
```

Comment:
may I know why we need this function?

Reply:
This function is used in [MoonbeamApi](https://github.com/AstarNetwork/astar-apps/blob/1b9ec9d950d7d8f1d3499db06dd0c337a2ab07de/src/hooks/xcm/parachainApi/MoonbeamApi.ts#L142). I think I have to define this in SubstarateAPI since MoonbeamApi extends from SubstarateAPI but please let me know if you have a better solution.